### PR TITLE
feat(core): add missing SO

### PR DESCRIPTION
**Pull request by stonebuzz**
Add missing SO for tasktemplate

Need for this-> https://github.com/pluginsGLPI/datainjection/pull/237

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | Internal 22192

### CODE REVIEW

File: inc/tasktemplate.class.php [hunk 1]
```diff
@@ -102,10 +102,101 @@ function rawSearchOptions() {
...
    }
 
 
+   /**
+   * @since 0.84
+   *
+   * @param $field
+   * @param $values
+   * @param $options   array
+   **/
```
Comment (trasher) [inline inc/tasktemplate.class.php:161]:
```suggestion
```

File: inc/tasktemplate.class.php [hunk 2]
```diff
@@ -102,10 +102,101 @@ function rawSearchOptions() {
...
+   /**
+    * @since 0.84
+    *
+    * @param $field
+    * @param $name            (default '')
+    * @param $values          (default '')
+    * @param $options   array
+    *
+    * @return string
+   **/
```
Comment (trasher) [inline inc/tasktemplate.class.php:185]:
```suggestion
```

File: inc/tasktemplate.class.php (hunk 1 above)
Comment (stonebuzz) [reply inc/tasktemplate.class.php]: cleaned

File: inc/tasktemplate.class.php (hunk 2 above)
Comment (stonebuzz) [reply inc/tasktemplate.class.php]: cleaned

**Review by orthagh — APPROVED** [review]

File: inc/tasktemplate.class.php [hunk 3]
```diff
@@ -102,10 +102,84 @@ function rawSearchOptions() {
...
+         'right'              => 'own_ticket'
+      ];
+
+      $tab[] = [
+         'id'                 => '8',
+         'table'              => 'glpi_groups',
+         'field'              => 'completename',
+         'linkfield'          => 'groups_id_tech',
+         'name'               => Group::getTypeName(1),
+         'condition'          => ['is_assign' => 1],
```
Comment (cedric-anne) [inline inc/tasktemplate.class.php:129]:
```suggestion
         'condition'          => ['is_task' => 1],
```

Comment (stonebuzz) [reply inc/tasktemplate.class.php:129]: done

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**